### PR TITLE
Convert FBX to PMO & PMP

### PR DESCRIPTION
**Pull request by Truthkey**
This Pull Request includes, amongst other things, conversion of FBX files to PMO and PMP.

PMO can only convert models without bones and not all features are currently supported.
Not supported:
- Bones
- Vertex Color

PMP is basically non-functional at this stage, but you can create basic 1-pmo PMP which should work.

*Example of KH2 model converted to PMO through FBX*
PMO_1

### CODE REVIEW

File: OpenKh.Command.PmpConverter/OpenKh.Command.PmpConverter.csproj
Comment (Topaz-Reality) [inline OpenKh.Command.PmpConverter/OpenKh.Command.PmpConverter.csproj:18]: Again, WinShell projects should not be referenced.

**Review by Topaz-Reality — CHANGES_REQUESTED** [review]

**Comment by Topaz-Reality** [comment]
Overall, I cannot see any other problems with your code. However, I will wait on someone else to approve as well after the issues I mentioned are fixed.

**Comment by Topaz-Reality** [comment]
A problem. It seems that some changes within your code causes the test ``OpenKh.Tests.Imaging.Tm2Tests.ValidateAllKH2MapRadarImages`` to fail with an ``InvalidDataException`` on ``Tm2.Read()``.
Can you look into that?

File: OpenKh.Command.PmoConverter/OpenKh.Command.PmoConverter.csproj
Comment (Truthkey) [reply OpenKh.Command.PmoConverter/OpenKh.Command.PmoConverter.csproj:18]: Oh yeah, I included it because I wanted to use the `MarshallBitmap` from it because I didn't know we had a `Png` class.
I'll remove those references in a second.

Comment (Topaz-Reality) [reply OpenKh.Command.PmoConverter/OpenKh.Command.PmoConverter.csproj]: Thanks for noticing and changing this on such short notice 👍

**Comment by Truthkey** [comment]
Made the necessary adjustments so it doesn't interfere with the way KH2 TM2s are read.

**Review by Topaz-Reality — APPROVED** [review]
Thanks for your contribution. Approved by me!

File: OpenKh.Imaging/Tm2.cs
Comment (Xeeynamo) [inline OpenKh.Imaging/Tm2.cs:390]: Here you could have used a `IImageRead` instead of a `PngImage`

**Comment by Xeeynamo** [comment]
I think that we need to add those tools in the [OpenKH Tool](https://openkh.dev/tool/) page.

File: OpenKh.Imaging/Tm2.cs
Comment (Truthkey) [reply OpenKh.Imaging/Tm2.cs:390]: I would have used it, but I don't know how. IImageRead is just an interface, so I don't know how I'm supposed to get any information other than `Size` and `PixelFormat`. Casting to PngImage?

Comment (Xeeynamo) [reply OpenKh.Imaging/Tm2.cs:390]: IImageRead has Size and PixelFormat. Those two properties you used from PngImage are actually from IImageRead interface. That way the `Tm2 Create` function can generate TM2 files from any other supported image formats in OpenKH!

Comment (Truthkey) [reply OpenKh.Imaging/Tm2.cs:390]: I just changed it to use IImageRead. Should be fine now, then.

Comment (Xeeynamo) [reply OpenKh.Imaging/Tm2.cs:390]: I think you have to create a new PR for that :'D

Comment (Topaz-Reality) [reply OpenKh.Imaging/Tm2.cs:390]: Sorry for missing this detail. I will ensure that it does not happen again ^^'

**Pull request merged**
